swarm/storage/mru/request.go

```diff
@@ -18,23 +18,15 @@ package mru
 
 import (
 	"bytes"
-	"encoding/binary"
 	"encoding/json"
 	"fmt"
-	"hash"
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
-	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/swarm/storage"
 )
 
-// Signature is an alias for a static byte array with the size of a signature
-const signatureLength = 65
-
-type Signature [signatureLength]byte
-
```

Comment:
Moved to `signedupdate.go`. In #704 is finally moved to `resource_sign.go`, as per @nolash's comments in the prior PR.